README.rst

```diff
@@ -214,8 +214,8 @@ Choose from the list of available rules:
 
   Configuration options:
 
-  - ``syntax`` (``'long'``, ``'short'``): whether to use the ``long`` or ``short`` array
-    syntax; defaults to ``'long'``
+  - ``syntax`` (``'long'``, ``'short'``): whether to use the ``long`` or ``short`` array syntax;
```

Comment:
http://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html#line-length

I guess we follow Silex/Twig/SF guidelines, however since we don't use sphinx (I think?) for generating docs we could drop the limit. So maybe a more generic question for the community; should we stay with `.rst` format or update to `.md` ?